opentext-contact-us-page.html

```diff
@@ -0,0 +1,116 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>Opentext Contact Us</title>
+        <style>
+            container {
+                max-width: 1200px;
+            }
+            .row {
+                clear: float;
+            }
+            .column {
+                width: 25%;
+                float: right;
+            }
+            ul {
+                list-style-type: none;
+            }
+            li {
+                display: inline;
+            }
+            li a {
+                display: inline;
+            }
+            .col-6 {
+                width: 16%;
```

Comment:
Just wondering why this is 16%?

Reply:
The “second navbar” has 6 columns. What is the naming convention for when there is two navbar?